osrsbox/items_api/stance.py

```diff
@@ -0,0 +1,43 @@
+from dataclasses import dataclass
+from enum import Enum
+from typing import Optional
+
+
+class AttackType(Enum):
+    STAB = "stab"
+    MAGIC = "magic"
+    DEFENSIVE_CASTING = "defensive casting"
+    SLASH = "slash"
+    RANGED = "ranged"
+    SPELLCASTING = "spellcasting"
+    CRUSH = "crush"
+    NONE = None
+    ALIAS_FOR_NONE = "none"
```

Comment:
I might update the `"none"` entries in the `data/weapon_stances.py` file. Then we can remove the `ALIAS_FOR_NONE` entry. I have manually modified a couple of entries in this file already, and think this should also be updated. The weapon attack types seem pretty static, so not too worried about updating some part manually.

I noticed there were two additional attack types for _Blasters_:

```
fires an explosive round
emits a jet of flame
```

I am guessing you did not include these on purpose? They do not seem that useful for anything.